### DISCUSSION TITLE
2024-01-29

### DISCUSSION THREAD

**Discussion by araujof** [sysflow-telemetry/sysflow#120] [Community meetings]
### Participants
- Luca Muscariello 
- Michele Papalini
- Mauro Sardara
- Teryl Taylor
- Fred Araujo

### Discussion

- Release management
  - Proposal to adopt release branches with hotfixes, etc. 
  - Releases should not be delayed based on documented, non-critical bugs
  - SysFlow major release cycle will track the Falco libs major release cycle
  - **Action**: Luca, Michele, and Mauro to draft the initial proposal
- Rules language support
  - Methodology/workflow to provide compatibility checks of our rules engine 
  - Compatibility level and differences to Falco version
    - "can sf-processor handle falco.yaml version 0.34?"
  - Tests for policy/language-level compatibility (black-box testing)
  - **Action**: TBD
- Multi-architecture support
  - Targets: ARM64, AMD64
  - Consider using GH's native runner for AMD64 [Luca]
- Longer-term issues
  - Move builds to make
- Community meetings:
  - Every Monday, 12 pm - 12:30 pm EST
  - **Action**: Fred: to create spaces in wiki for minutes, new proposals, etc.

**Comment by muscariello** [comment]
On arm64 GH hosted runners:
These runners are available to those using the Team or Enterprise Cloud plans.
https://resources.github.com/devops/accelerate-your-cicd-with-arm-and-gpu-runners-in-github-actions/
